src/lib/support/UnitTestRegistration.cpp

```diff
@@ -22,7 +22,7 @@
 
 namespace chip {
 
-const size_t kTestSuitesMax = 128;
+const size_t kTestSuitesMax = 256;
```

Comment:
Why did the max # of test suites need to be increased?
I'd have thought this max was per test executable.

Reply:
The nrf test suite was running out of space in the array and was crashing. @Damian-Nordic suggested i increase the limit since the nrf suite runs all the unit tests at once.